dynamicpage.js

```diff
@@ -0,0 +1,250 @@
+            </div>
+
+          </div>
+
+          <div class="column2 ">
+            <i><img class="speaker" src="${data[4].image}" alt=""></i>
+            <div class="speaker-content">
+              <h2>${data[4].name}</h2>
+              <p class="about-title2">${data[4].title}</p>
+              <hr>
+              <p class="about-content">${data[4].description}</p>
+            </div>
+
+          </div>
+
+          <div class="column2 ">
+            <i><img class="speaker" src="${data[5].image}" alt=""></i>
+            <div class="speaker-content">
+              <h2>${data[5].name}</h2>
+              <p class="about-title2">${data[5].title}</p>
+              <hr>
+              <p class="about-content">${data[5].description}</p>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+
+
+
+</div>
+
+</div>
+
+
+
+<div class="more-desktop">
+<div class="container text-enter about-main-programs">
+
+  <div class="speaker-content1">
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[0].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[0].name}</h2>
+        <p class="about-title2">${data[0].title}</p>
+        <hr>
+        <p class="about-content">${data[0].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[1].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[1].name}</h2>
+        <p class="about-title2">${data[1].title} </p>
+        <hr>
+        <p class="about-content">${data[1].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[2].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[2].name}</h2>
+        <p class="about-title2">${data[2].title}</p>
+        <hr>
+        <p class="about-content">${data[2].description}</p>
+      </div>
+
+    </div>
+  </div>
+
+  <div class="speaker-content2">
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[3].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[3].name}</h2>
+        <p class="about-title2">${data[3].title}</p>
+        <hr>
+        <p class="about-content">${data[3].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[4].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[4].name}</h2>
+        <p class="about-title2">${data[4].title}</p>
+        <hr>
+        <p class="about-content">${data[4].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[5].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[5].name}</h2>
+        <p class="about-title2">${data[5].title}</p>
+        <hr>
+        <p class="about-content">${data[5].description}</p>
+      </div>
+
+    </div>
+  </div>`;
```

Comment:
- [ ] I can see you added a function that iterates through the speaker's list on the window load good job so far 👍  but there is something that you are still missing out. Kindly consider removing the hard-coded way to  follow [JS best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) as the function  iterates in the speaker's array and for each it creates that single Html

Reply:
Is this ok?

![image](https://user-images.githubusercontent.com/111823606/233483104-cc77bfd7-b507-41ad-a5a1-267f43bd09dc.png)